docs/source/core_concepts/er_model.rst

```diff
@@ -0,0 +1,83 @@
+Entity-relationship model
+=========================
+
+Each Hyperledger Iroha peer has a state, called "World State View",
+which is represented by a set of entities and relations between them.
+To explain you more which entities exist in the system and what are the relations,
+this sections includes ER diagram and an explanation for its components.
+
+ER diagram
+----------
+
+.. image:: ./../../image_assets/er-model.png
+
+Peer
+----
+
+ - address — network address and internal port, is used for synchronization, consensus, and communication with the ordering service
+ - public_key — key, which will be used for signing blocks during consensus process
+
+Asset
+-----
+
+ - asset_id — identifier of asset, formatted as asset_name#domain_id
+ - domain_id — identifier of domain, where the asset was created, references existing domain
+ - precision — size of fractional part
+ - data — JSON with arbitrary structure of asset description
+
+Signatory
+---------
+
+ - public_key — a public key
+
+Domain
+------
+
+ - domain_id — identifier of a domain
+ - default_role — a default role per user created in the domain, references existing role
```

Comment:
Again, a stylish issue - this is the only entry in ER-diagram, which is written with a smaller font. Extending the Domain rectangle by a couple of pixels will make it perfect

Reply:
Default export from workbench…